packtools/sps/formats/oai_dc_agris.py

```diff
@@ -0,0 +1,689 @@
+                xmlns:agls="http://www.naa.gov.au/recordkeeping/gov_online/agls/1.2"
+                xmlns:dcterms="http://purl.org/dc/terms/">
+                    <ags:resource ags:ARN="XS2021000111">
+            </metadata>
+        </record>
+    """
+    el = ET.Element('{http://purl.org/agmes/1.1/}resource')
+    el.set('{http://purl.org/agmes/1.1/}ARN', 'XS' + get_identifier(xml_tree))
+
+    xml_oai_dc_agris.find('./metadata/{http://purl.org/agmes/1.1/}resources').append(el)
+
+
+def xml_oai_dc_agris_title_pipe(xml_oai_dc_agris, xml_tree):
+    """
+    Schema (https://agris.fao.org/agris_ods/dlio.dtd.txt):
+        <!-- ELEMENT title -->
+        <!ELEMENT dc:title (#PCDATA | dcterms:alternative | ags:titleSupplement)*>
+        <!ATTLIST dc:title
+            xml:lang CDATA #IMPLIED
+        >
+
+    Example:
+        <record>
+            <metadata>
+                <ags:resources>
+                    <ags:resource ags:ARN="XS2021000111">
+                        <dc:title xml:lang="es">
+                            <![CDATA[ La canción reflexiva: en torno al estatuto crítico de la música popular en Brasil ]]>
+                        </dc:title>
+                    </ags:resource>
+                </ags:resources>
+            </metadata>
+        </record>
+    """
+    add_title(xml_oai_dc_agris, xml_tree)
+
+
+def xml_oai_dc_agris_creator_pipe(xml_oai_dc_agris, xml_tree):
+    """
+    Schema (https://agris.fao.org/agris_ods/dlio.dtd.txt):
+        <!-- ELEMENT creator -->
+        <!ELEMENT dc:creator (#PCDATA | ags:creatorPersonal | ags:creatorCorporate | ags:creatorConference)*>
+        <!ELEMENT ags:creatorPersonal (#PCDATA)>
+        <!ELEMENT ags:creatorCorporate (#PCDATA)>
+        <!ELEMENT ags:creatorConference (#PCDATA)>
+
+    Example:
+        <record>
+            <metadata>
+                <ags:resources>
+                    <ags:resource ags:ARN="XS2021000111">
+                        <dc:creator>
+                            <ags:creatorPersonal>de-Oliveira-Gerolamo, Ismael</ags:creatorPersonal>
+                        </dc:creator>
+                    </ags:resource>
+                </ags:resources>
+            </metadata>
+        </record>
+    """
+    author = article_authors.Authors(xml_tree)
+    try:
+        surname = author.contribs[0].get('surname')
+        given_name = author.contribs[0].get('given_names')
+        author_name = f' {surname.strip()}, {given_name.strip()} '
+        add_creator(xml_oai_dc_agris, author_name.strip())
+    except IndexError:
+        pass
+
+
+def add_publisher(xml_oai_dc_agris, publisher):
+    try:
+        ags = ET.Element('{http://purl.org/agmes/1.1/}publisherName')
+        ags.text = publisher.publishers_names[0].strip()
+
+        dc = ET.Element('{http://purl.org/dc/elements/1.1/}publisher')
+        dc.append(ags)
+
+        xml_oai_dc_agris.find(".//ags:resource", {"ags": "http://purl.org/agmes/1.1/"}).append(dc)
+    except IndexError:
+        pass
```

Comment:
@Rossi-Luciano se não existir publisher é OK? Se não sugiro criar a exceção AddPublisherError e levantá-la no lugar de `pass`

Reply:
@robertatakenaka, pelo que pude compreender, quando há o `'*'` significa nenhum ou muitos e quando há `'+'` significa um, pelo menos. No caso de publisher a seguinte descrição aparece com '*', veja: 

```
Schema (https://agris.fao.org/agris_ods/dlio.dtd.txt):
        <!-- ELEMENT publisher -->
        <!ELEMENT dc:publisher (ags:publisherName | ags:publisherPlace)*>
        <!ELEMENT ags:publisherName (#PCDATA)>
        <!ELEMENT ags:publisherPlace (#PCDATA)>
```
Mas não estou seguro disso, posso inserir o tratamento de exceção correto, se for o caso.